.github/workflows/failureNotifier.yml

```
@@ -50,14 +83,9 @@ jobs:
               if (jobs.jobs[i].conclusion == 'failure') {
                 const jobName = jobs.jobs[i].name;
                 const jobLink = jobs.jobs[i].html_url;
-                const issues = await github.rest.issues.listForRepo({
-                  owner: context.repo.owner,
-                  repo: context.repo.repo,
-                  labels: failureLabel,
-                  state: 'open'
-                });
-                const existingIssue = issues.data.find(issue => issue.title.includes(jobName));
-                if (!existingIssue) {
+                const previousJob = previousRunJobs.jobs.find(job => job.name === jobName);
+                previousJobSucceeded = previousJob.conclusion === 'success';
```

```suggestion
                previousJobSucceeded = previousJob?.conclusion === 'success';
```

I guess it's possible that `previousJob` could return false for some reason.